packages/flutter_tools/lib/executable.dart

```diff
@@ -59,6 +59,7 @@ import 'src/usage.dart';
 ///
 /// This function is intended to be used from the `flutter` command line tool.
 Future<Null> main(List<String> args) async {
+  SigintProcessSignal.instance.init();
```

Comment:
It's important that the default signal handler gets registered even if nobody watches this event. Otherwise the work-around for not breaking the windows console when Ctrl+C is pressed (the whole point of this exercise) wouldn't work.